firecrown/ccl/two_point.py

```diff
@@ -7,6 +7,8 @@
     _parse_two_point_statistics,
     _parse_likelihood)
 
+from ._ccl import compute_loglike, write_stats  # noqa
```

Comment:
Why is compute_loglike moving within this PR?

Reply:
To enable a new ccl api that allows for both clusters and two point stats.